src/constants.py

```diff
@@ -20,6 +20,10 @@
 WORKLOAD_OS_GROUP = "postgres"
 WORKLOAD_OS_USER = "postgres"
 METRICS_PORT = "9187"
-POSTGRES_LOG_FILES = ["/var/log/pgbackrest", "/var/log/postgresql"]
+POSTGRES_LOG_FILES = [
+    "/var/log/pgbackrest",
```

Comment:
I'm not sure how pgbackrest calls its log files

Reply:
Hmm, I saw different logs related to backups, restores and stanza initialisation, but I don't know the complete list, so if the directory works here, that's fine.

Reply:
I have feeling the folder doesn't work. But let's test it this way.